modules/ROOT/pages/maven-preferences-reference.adoc

```diff
@@ -39,11 +49,57 @@ Selecting this option requires you to define your custom path to your Maven loca
 | Empty
 
 | Base command line for builds
-| The Maven command to execute every time you run your Mule project in your local machine.
+| The Maven command to execute every time you build or run a Mule project in your local machine.
 | mvn clean package -nsu -DskipMunitTests
 
 |===
 
+== Configure Global Settings
+
+Studio manages the servers used in the global settings and you can configure Studio to use an external Maven installation. By doing so, Studio automatically detects and populates the IDs from the *settings.xml* file stored in *${maven.installation.home}/conf/settings.xml*.
+
+image::maven-global-settings.png[“*Global Settings* option highlighted in the Studio setting preferences window.”]
+
+If there is no file on this path, you can create a new *settings.xml* file in other location:
+
+. Select *Override global settings path*.
+. Click *Browse*.
+. Select your *settings.xml* file and click *Open*.
++ The IDs are listed.
+. Click *Apply and Close*.
+
+== Configure Update Policies
+
+You can configure Maven update policies depending on your needs.
+
+image::maven-update-policies.png[“*Update Policies* option highlighted in the Studio setting preferences window.”]
+
+More information about https://maven.apache.org/settings.html#repositories[Update Policies^] in Maven
+
+== Configure User Settings
+
+Studio manages the servers for downloading dependencies or plugins and defines the credentials of such servers if needed. It automatically detects and populates the IDs from the *settings.xml* file located in *${user.home}/.m2/settings.xml*.
+
+image::maven-user-settings.png[“*User Settings* option highlighted in the Studio setting preferences window.”]
+
+If there is no file on this path, you can create a new *settings.xml* file in other location:
+
+. Select *Override user settings path*.
+. Click *Browse*.
+. Select your *settings.xml* file and click *Open*.
++ The IDs are listed.
+. Click *Apply and Close*.
+
+=== Add a Server
+
+You can add a new server to your *settings.xml* file: 
+
+. In the *User Settings* window, click *Add*.
+. Enter the server Id and, your username and password.
```

Comment:
```suggestion
. Enter the server ID and, your username and password.
```

Reply:
Great catch on the "ID"! Thanks!
Regarding the comma, I think it is needed to avoid misunderstandings in the grammar structure of the sentence.
The reader should enter two blocks of information, and the second block has two items, therefore the comma is needed:
"Enter (1) the server ID and, (2) your (2.1) username and (2.2) password".